lib/as.js

```diff
@@ -10,9 +10,9 @@ function asBuffer(body, options) {
   if (Buffer.isBuffer(body)) {
     ret = body;
   } else if (typeof body === 'object') {
-    ret = new Buffer(JSON.stringify(body), options.reqBodyEncoding);
+    ret = Buffer.from(JSON.stringify(body), options.reqBodyEncoding);
```

Comment:
Buffer constructor is deprecated and the `from` method is available in node v6+ so it shouldn't be a breaking change. Fixed it to remove the warnings during the tests